routes/boxHistory.js

```diff
@@ -1,21 +1,63 @@
 const express = require('express');
-const {
-  getBoxByID,
-  getBoxesWithStatusOrPickup,
-  updateBox,
-  approveBoxInBoxHistory,
-  copyBoxInfoToAnchorBox,
-} = require('../services/boxHistoryService');
 
 const boxRouter = express();
+const pgp = require('pg-promise')({});
+const { db } = require('../config');
+
+const cn = `postgresql://${process.env.REACT_APP_DATABASE_USER}:${process.env.REACT_APP_DATABASE_PASSWORD}@${process.env.REACT_APP_DATABASE_HOST}:${process.env.REACT_APP_DATABASE_PORT}/${process.env.REACT_APP_DATABASE_NAME}?ssl=true`; // For pgp
+
+const database = pgp(cn);
+
 boxRouter.use(express.json());
 
+const SQLQueries = {
```

Comment:
Please merge dev and refactor your added queries to match the route-controller-service method. Here is an example - https://github.com/ctc-uci/find-your-anchor-backend/pull/15/files